### ISSUE TITLE
Possible enhancement. Skip through Tapping 

### ISSUE THREAD

**Issue by rasselll** [abbasfreestyle/react-native-af-video-player#31]
Hey, I was wonder could you add a double tap on the left/right sides of the video player, that would skip/rewind the video 5 second

**Comment by EddieEJ** [comment]
> Thanks, i'll add it in the to-do list and see if it's quick to implement.

@abbasfreestyle Has there been any progress regarding this feature? Personally, I think this is a great idea and is a widely-used functionality(mostly in YouTube) and would make this wonderful player even better!

**Comment by uday5162** [comment]
@EddieEJ Yes this functionality is widely used and as time has progressed from the question can i know how you have implemented this.

**Comment by uday5162** [comment]
after some research I have implemented this functionality. Here is the pull request

https://github.com/abbasfreestyle/react-native-af-video-player/pull/142#issue-337971619
